Assets/Mirror/Runtime/Transport/Websocket/Server.cs

```diff
@@ -232,15 +237,15 @@ async Task<ArraySegment<byte>> ReadFrames(int connectionId, WebSocketReceiveResu
 
             while (!result.EndOfMessage)
             {
-                if (count >= MaxMessageSize)
+                if (count >= _maxMessageSize)
```

Comment:
```suggestion
                if (count > _maxMessageSize)
```
it is ok if count == _maxMessageSize